starboard/android/shared/media_capabilities_cache.cc

```diff
@@ -231,23 +231,9 @@ bool VideoCodecCapability::IsBitrateSupported(int bitrate) const {
   return supported_bitrates_.Contains(bitrate);
 }
 
-bool VideoCodecCapability::AreResolutionAndRateSupported(
-    bool force_improved_support_check,
-    int frame_width,
-    int frame_height,
-    int fps) {
-  if (force_improved_support_check) {
```

Comment:
`force_improved_support_check` is on by default (check line 87 in media_is_video_supported.cc). If we're removing it, we still need to keep this logic to maintain the default behavior.

Reply:
Remove `force_improved_support_check ` and keep the logic inside the statement.